docs/sdk-configuration.md

```diff
@@ -10,7 +10,7 @@ guide for those that have support already.
 | --- | --------- | ------- | ------- |
 | [Java AWS SDK (JVM)](#java-aws-sdk-jvm) | :heavy_check_mark: | `>=1.11.394` | |
 | [Python AWS SDK (boto3)](#python-aws-sdk-boto3) | :heavy_check_mark: | `>=1.9.28` | |
-| [AWS CLI](#aws-cli) | :heavy_check_mark: | `>=1.16.43` | |
+| [AWS CLI](#aws-cli) | :heavy_check_mark: | `>=1.16.43`, `>=2.7.8` | Configuration differs slightly between v1 and v2 of the SDK |
```

Comment:
This seems to be in the wrong row. Should be in the First row above with "Java AWS SDK".